asset/snippets/test/samples.test.js

```diff
@@ -278,37 +284,29 @@ describe('quickstart sample tests', () => {
 
   it('should create saved query successfully', async () => {
     const description = 'description';
-    const stdout = execSync(
-      `node createSavedQuery.js ${queryId} ${description}`
-    );
-    assert.include(stdout, `${savedQueryFullName}`);
+    const result = await createSavedQuery(queryId, description);
+    assert.include(result.name, `${savedQueryFullName}`);
```

Comment:
**Nit:** we don't need to wrap variables that are already strings in the `${...}` operator.
```suggestion
assert.include(result.name, savedQueryFullName);
```

_Please apply this comment throughout your PR - thanks! 🙂_

Reply:
applied:)